modules/ROOT/pages/publish-apis.adoc

```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-As an Anypoint API Community Manager community administrator, you can add and publish APIs and minor and major updates from Anypoint Exchange into your community. 
+As an Anypoint API Community Manager community administrator, you can add and publish APIs and API Groups and minor and major updates from Anypoint Exchange into your community. 
```

Comment:
As an Anypoint API Community Manager community administrator, you can add and publish APIs and API Groups and minor and major updates from Anypoint Exchange into your community. 
I'm thinking this is the first reference to ACM that is why you are going the whole nine yards. If not, you can do ACM right?
As an Anypoint API Community Manager community administrator, you can publish (updated API Groups or) updates API Groups from Anypoint Exchange into your community?

Reply:
Correct, this is the first reference to ACM.

Updated the text for clarity.